cabal-install/tests/IntegrationTests2.hs

```diff
@@ -1729,6 +1729,7 @@ planProject testdir cliConfig = do
                          distDirLayout cabalDirLayout
                          projectConfig
                          localPackages
+                         Nothing
```

Comment:
This tests the new behaviour or it was edded to make it compile?
Afaiu a test checking the env file expected content would be needed

Reply:
I agree adding a test would be good. This indeed just makes it compile.

Reply:
However -- a regression test against this behavior is _not_ the test needed. The behavior was something complicated that was  removed, its not just going to magically reappear. What we need are some standard integration tests for "cabal v2-install" _at all_.

Reply:
Yeah i was thinking in generating a env file with the version of this pr and test against a golden file (or regexp-like checks testing the relevant parts)
And well regression test goal is just that, ensure the behaviour, well, does not regress, magically or by accident 😄